eZ/Publish/Core/FieldType/Date/SearchField.php

```diff
@@ -83,4 +83,40 @@ public function getDefaultSortField()
     {
         return $this->getDefaultMatchField();
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFullTextData(Field $field, FieldDefinition $fieldDefinition)
+    {
+        // The field type stores date value as a timestamp of the start of the day in the
+        // environment's timezone.
+        // We format this as Y-m-d and add Z to signify UTC (zero offset).
+        $dateTime = new DateTime();
+        $dateTime->setTimestamp($field->value->data['timestamp']);
+
+        return [
+            $dateTime->format('Y-m-d\\Z'),
```

Comment:
True, fixed in 06e437d. By integration test did you mean test checking that this does not return FullText or something else?